### ISSUE TITLE
Ability to comment on Origin Trial Privacy, Security, Debuggability, API Owners reviews

### ISSUE THREAD

**Issue by LuHuangMSFT** [GoogleChrome/chromium-dashboard#3453]
**Describe the bug**
The Origin Trial section of the page for managing features has 4 buttons (Privacy, Security, Debuggability, API Owners) that open a side panel. I should be able to add comments to each of the reviews in the side panel but no entry box is available.

**To Reproduce**
1. Go to https://chromestatus.com/feature/5746537956114432?context=myfeatures
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/53061947/62b62454-9162-4d03-94ce-3af499c72fc9)

**Comment by jrobbins** [comment]
Also, I don't believe that you were a registered user of the app.  I just added you as a registered user.

**Comment by LuHuangMSFT** [comment]
Yes, I am logged in with an account that's listed as one of the feature owners.
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/53061947/db50ca6c-b5cd-4a96-9c17-37b0eb16223d)
![image](https://github.com/GoogleChrome/chromium-dashboard/assets/53061947/2e264aaa-b9a4-44ad-b97c-974441a11196)

**Comment by LuHuangMSFT** [comment]
Signed out and signed back in. Still unable to add comments.

**Comment by jrobbins** [comment]
Sorry about that.  Try reloading the page now.
I had added an incorrect variation of your email address.
